feature1/one-more-file.txt

```diff
@@ -0,0 +1,3 @@
+This file was also added in the feature-1 branch.
```

Comment:
It's really good, that you have decided on this.

Reply:
Thanks for the comment. Zeolith.